src/DynamicEvaluator/Application/Checker/ClassExtendsUpdatedPackageChecker/Dto/ViolationDto.php

```diff
@@ -7,9 +7,11 @@
 
 declare(strict_types=1);
 
-namespace Upgrade\Application\Dto;
+namespace DynamicEvaluator\Application\Checker\ClassExtendsUpdatedPackageChecker\Dto;
 
-class ViolationDto
+use Upgrade\Application\Dto\ViolationDtoInterface;
+
+class ViolationDto implements ViolationDtoInterface
```

Comment:
The second Violation with an equal name. it's hard to read in code when you use them

Reply:
Same as above